src/app/pages/CpsPgl/index.jsx

```diff
@@ -25,6 +25,7 @@ import visuallyHiddenHeadline from '#containers/VisuallyHiddenHeadline';
 import {
   getFirstPublished,
   getLastPublished,
+  getAboutTags,
 } from '#containers/ArticleMain/utils';
```

Comment:
Perhaps we should think about moving these utilities out of the `ArticleMain` scope and placing them inside `src/lib/utilities`.

To me, this currently indicates that this utility file is only used within `ArticleMain`.

Reply:
@12 , I agree with this refactor personally, but don't want to hold up the merge of this issue on this point alone. Do you fancy raising a follow-up PR for discussion?

Reply:
@andrewscfc Have raised #5245